resources/definitions/Infinity3D.def.json

```
@@ -0,0 +1,77 @@
+{
+    "version": 1,
+    "name": "Infinity3D",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": false,
+        "manufacturer": "Infinity3D",
+        "file_formats": "text/x-gcode",
+        "machine_extruder_trains": {
+            "0": "Infinity3D_extruder_0"
+        },
+        "has_materials": true,
+        "has_machine_quality": true,
+        "preferred_quality_type": "normal",
+        "preferred_material": "generic_pla",
+        "exclude_materials": [ ]
```

```suggestion
        "preferred_material": "generic_pla"
```
DRY Principle. No need to overide settings with their default